src/Tizen.Sensor/Tizen.Sensor/BatchData/HeartRateMonitorBatchData.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Tizen.Sensor.BatchData
```

Comment:
I think, it is unnecessary namespace
```suggestion
namespace Tizen.Sensor
```